inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -217,6 +217,21 @@ class V10Parser : public IParser {
                 std::vector<size_t> shape;
                 if (!getParameters<size_t>(node.child("data"), name, shape)) return;
                 static_cast<ngraph::Strides&>(*a) = ngraph::Strides(shape);
+#ifdef __APPLE__
```

Comment:
Why do we need this platform specific code?
Can we use set or static_cast for both cases?

Reply:
No, it doesn't work. The compilation error is thrown if I use one function. In the ngraph/core/include/ngraph/attribute_adapter.hpp in line 371 we have another example of similar issue.